README.md

```diff
@@ -7,6 +7,19 @@
 [![Discord][discord-shield]][discord-url]
 
 
+# Kontent.ai React Integration Template
+
+This template can be used to jumpstart your own integration development with Kontent.ai. It contains all the neccessary tools for creating a new [Custom Element](https://kontent.ai/learn/tutorials/develop-apps/integrate/content-editing-extensions/), a UI extension for content editors. 
+
```

Comment:
Can we add the features? 

* Parent frame check
* Height adjustments
* OnDisabled handling
* Watched wlement
* Input element
* Selected assets
* Selected Items